ws-client/src/client.rs

```diff
@@ -394,10 +389,11 @@ fn process_response(
 		RequestStatus::PendingMethodCall => {
 			let send_back_oneshot = manager.complete_pending_call(response_id).ok_or(Error::InvalidRequestId)?;
 			let response = response.try_into().map_err(Error::Request);
-			match send_back_oneshot.send(response) {
-				Err(Err(e)) => Err(e),
-				Err(Ok(_)) => Err(Error::Custom("Frontend channel closed".into())),
-				Ok(_) => Ok(None),
+			match send_back_oneshot.map(|tx| tx.send(response)) {
+				Some(Err(Err(e))) => Err(e),
+				Some(Err(Ok(_))) => Err(Error::Custom("Frontend channel closed".into())),
+				Some(Ok(_)) => Ok(None),
+				None => Ok(None),
```

Comment:
I added that `send_back` can be `None` because subscriptions are different from ordinary requests and the actual `drop imp` doesn't care about the response.